lib/vmpooler/pool_manager.rb

```diff
@@ -496,33 +496,16 @@ def purge_unused_vms_and_folders
       nil
     end
 
-    # Return a list of pool folders
-    def pool_folders(provider_name)
-      folders = {}
-      $config[:pools].each do |pool|
-        next unless pool['provider'] == provider_name.to_s
-
-        folder_parts = pool['folder'].split('/')
-        datacenter = $providers[provider_name.to_s].get_target_datacenter_from_config(pool['name'])
-        folders[folder_parts.pop] = "#{datacenter}/vm/#{folder_parts.join('/')}"
-      end
-      folders
-    end
-
-    def get_base_folders(folders)
-      base = []
-      folders.each do |_key, value|
-        base << value
-      end
-      base.uniq
-    end
-
     def purge_vms_and_folders(provider_name)
```

Comment:
Does the name need to have `folders` removed from it?

Reply:
sure, thats all internal and part of the main loop, so a remane should not affect anything, but there is a cascade of methods that need renamed, I'll do that now

Reply:
Thanks. I think it will help with making the code easier for others to understand.